doc/ref_arch/kubernetes/chapters/chapter05.rst

```diff
@@ -41,26 +38,26 @@ come into picture that govern the security. Each of these layers needs to be
 secured within their perimeters. The various layers that come into picture are:
 
 -  **Container Registry**: A container registry is a repository to manage
-   container \**images. The access to container registry needs to be secured in
-   order to \**prevent unauthorised access or image tampering.
+   container images. The access to container registry needs to be secured in
+   order to prevent unauthorised access or image tampering.
 -  **Container Images**: Stored instance of a container that holds a set of
-   software needed to run an application. Before loading them to container
+   software needed to run an application. Before loading them to a container
    registry, they need to be secured by performing various checks like
    vulnerability analysis, scans etc. These images should also be signed from
    trusted sources.
 -  **Containers**: A lightweight and portable executable image that contains
    software and all of its dependencies. The containers need to be prevented from
    accessing the underlying OS like loading of kernel modules, mounting of
-   directories of underlying OS etc and ensuring that they don't run in
+   directories of underlying OS etc and it must be ensured that containers don't run in
```

Comment:
tense has changed from what precedes.  The new changes read:
**Containers**: A lightweight and portable executable image that contains software and all of its dependencies. The containers need to be prevented from accessing the underlying OS like loading of kernel modules, mounting of directories of underlying OS etc and it must be ensured that containers don't run in privileged mode.

Suggest
**Containers**: A lightweight and portable executable image that contains software and all its dependencies. The containers need to be prevented from accessing the underlying OS capabilities like loading of kernel modules, mounting of directories of the underlying OS, etc., and running in privileged mode.

```suggestion
   directories of the underlying OS, etc., and running in
```